sdk/media/azure-media-analytics-edge/setup.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import sys
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-media-analytics-edge"
+NAMESPACE_NAME = "azure.media.analyticsedge"
+PACKAGE_PPRINT_NAME = "Azure Media Live Video Analytics Edge SDK"
+
+# a-b-c => a/b/c
+package_folder_path = NAMESPACE_NAME.replace('.', '/')
+
+# azure v0.x is not compatible with this package
+# azure v0.x used to have a __version__ attribute (newer versions don't)
+try:
+    import azure
+    try:
+        ver = azure.__version__
+        raise Exception(
+            'This package is incompatible with azure=={}. '.format(ver) +
+            'Uninstall it with "pip uninstall azure".'
+        )
+    except AttributeError:
+        pass
+except ImportError:
+    pass
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '_version.py'), 'r') as fd:
+    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
+                        fd.read(), re.MULTILINE).group(1)
+
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.md', encoding='utf-8') as f:
+    readme = f.read()
+with open('CHANGELOG.md', encoding='utf-8') as f:
+    changelog = f.read()
+
+exclude_packages = [
+        'tests',
+        'tests.*',
+        'samples',
+        # Exclude packages that will be covered by PEP420 or nspkg
+        'azure',
+    ]
+if sys.version_info < (3, 5, 3):
```

Comment:
That's not allowed, please remove that if. If you have problem without that diff, please let me know why. If you saw that somewhere else, please let me know where so I can fix it (it's REALLY badly wrong)